IdFilter.java

```diff
@@ -0,0 +1,19 @@
+package com.google.test;
+
+import java.util.Collection;
+
+/**
+ * Filters hardcoded test accounts out of a collection of IDs.
+ * The test accounts are 100, 101 and 110.
+ */
+public class IdFilter {
+  public static boolean containsTestAccount(Collection<Long> ids) {
+    return ids.contains(100) || ids.contains(101) || ids.contains(110);
+  }
+
+  public static void filter(Collection<Long> ids) {
+    ids.remove(100);
```

Comment:
[CollectionIncompatibleType] Argument '100' should not be passed to this method; its type int is not compatible with its collection's type argument Long